src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesManagement.Codeunit.al

```diff
@@ -518,6 +561,8 @@ codeunit 396 NoSeriesManagement
         ReplaceNoText(No, NewNo, 0, StartPos, EndPos);
     end;
 
+#if not CLEAN24
+    [Obsolete('The method was moved to No. Series Setup Impl. and is now internal.', '24.0')]
```

Comment:
The Obsolete text does not match the content:
- The method is still available (was not moved yet).
- Within the procedure, I see no reference to the `No. Series Setup Impl.` yet.

Reply:
As with all obsoletions, the old method is still there because otherwise it would be breaking and also, you wouldn't get to see the obsolete warning. Now, we could change all these to state: The method will be moved, etc... but that's also not true. I could say the mehtod was copied and the old one will be removed. 
As for the second comment, I intend to keep the old code as unchanged as possible to reduce the risk of accidental breaks. 

Now, since i wrote this I added a facade and added the method to the facade so it is no longer internal...